micov/cli.py

```diff
@@ -105,7 +111,12 @@ def compress(data, output, disable_compression, lengths):
         df.write_csv(output, separator='\t', include_header=True)
     else:
         genome_coverage = coverage_percent(df, lengths).collect()
-        genome_coverage.write_csv(output, separator='\t', include_header=True)
+
+        if taxonomy is None:
+            genome_coverage.write_csv(output, separator='\t', include_header=True)
+        else:
+            genome_coverage_with_taxonomy = set_taxonomy_as_id(genome_coverage, taxonomy)
```

Comment:
Oh I see, the lineage is the ID. I think it's important we retain the genome ID itself because the lineages are not assured to be unique -- there are many E coli genomes for example